### PR TITLE
Added expert settings

### PR DESCRIPTION
**Pull request by williandalton**
@hliebscher
Added A_CYC_RH_LEVEL_MODE (Feuchtigkeit) and A_CYC_SUPPLY_HEATING_ADJUST_MODE (Methoden der Temperaturregelung) to config.js

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -376,7 +415,7 @@ function create_iobroker_object(id, role, type, dataType, states, writable, text
 }
 
 function determine_unit(key) {
-    if (key.indexOf("_TEMP_") > 0)
+    if (key.indexOf("_TEMP_") > 0 && key.indexOf("_MODE") < 0)
```
Comment (hacki11) [inline lib/config.js]: i think we should eliminate that function completely and provide the unit for each object. I understand your workaround, if we do not refactor that in this pull request we should do it in another in near future.

**Comment by hacki11** [comment]
Thanks for the contribution! 
Please give me a ping if you are ready to merge, i assume the commented state is waiting to be completed with @hliebscher s info

**Comment by hliebscher** [comment]
The Added A_CYC_RH_LEVEL_MODE (Feuchtigkeit) and A_CYC_SUPPLY_HEATING_ADJUST_MODE works fine for 270MV 
For A_CYC_OPT_TEMP_SENSOR_MODE we have to test with sensor. 
without we get "-273" as return value

File: lib/config.js (hunk 1 above)
Comment (williandalton) [reply lib/config.js]: I also think an unit per object would be the best solution.

Unfortunatly the workaround doesn´t fix the problem, maybe you´ve got another idea for a quick solution? VlxDevConstants.C_CYC_OPT_NONE is still always changed to -273

If not, we should leave A_CYC_OPT_TEMP_SENSOR_MODE commented out (as it is the pull request) and add it when we support units.

**Comment by hacki11** [comment]
There is a similar solution within vallox-api. Perhaps it does calculate the Kelvin offset on any _Temp_ value like I do with the unit.
I will look into it tonight tomorrow. I think we can fix that. 
@danielbayerlein

**Comment by williandalton** [comment]
Added new pull Request with all 3 States and without "not working workaround"

https://github.com/hacki11/ioBroker.valloxmv/pull/13

**Pull request closed without merging**